### PR TITLE
COMMONS-476 : Ability to disable the BloomFilter via property configu…

### PR DESCRIPTION
**Pull request by aboughzela**
COMMONS-476 : Ability to disable the BloomFilter via property configuration.

### CODE REVIEW

File: commons-extension-webapp/src/main/webapp/WEB-INF/conf/commons-extension/jcr/repository-configuration.xml [hunk 1]
```diff
@@ -36,6 +36,7 @@
 							<property name="update-storage" value="true"/>
 							<property name="max-buffer-size" value="204800"/>
 							<property name="swap-directory" value="${gatein.jcr.data.dir}/swap/system${container.name.suffix}"/>
+							<property name="acl-bloomfilter-enabled" value="${gatein.jcr.bloomfilter.enabled:true}"/>
```
Comment (thomasdelhomenie) [inline commons-extension-webapp/src/main/webapp/WEB-INF/conf/commons-extension/jcr/repository-configuration.xml]: Properties should start by "exo.", it should be "exo.jcr.bloomfilter.enabled".

Comment (aboughzela) [inline commons-extension-webapp/src/main/webapp/WEB-INF/conf/commons-extension/jcr/repository-configuration.xml]: We should use the same rule than the other properties used in repository-configuration.
(gatein.jcr.datasource.nam, gatein.jcr.datasource.dialect, gatein.jcr.storage.enabled ...)
Properties should start by "exo." : it's make sense to review all property name dosen't respect this rule. WDYT ?

**Comment by fdrouet** [comment]
With your current proposal @aboughzela we can't disable bloom filter on a single workspace like we are currently doing in production. Please provide a finer grain configuration

File: commons-extension-webapp/src/main/webapp/WEB-INF/conf/commons-extension/jcr/repository-configuration.xml (hunk 1 above)
Comment (thomasdelhomenie) [inline commons-extension-webapp/src/main/webapp/WEB-INF/conf/commons-extension/jcr/repository-configuration.xml]: No, it is not the purpose of this issue to rename all properties. So we should instead do the same thing than for the others properties : keep your value (gatein.jcr.bloomfilter.enabled), and map this property to "exo.jcr.bloomfilter.enabled" in https://github.com/exodev/platform/blob/develop/extension/config/src/main/resources/conf/platform/configuration.properties (like https://github.com/exodev/platform/blob/develop/extension/config/src/main/resources/conf/platform/configuration.properties#L833 for example).

**Comment by aboughzela** [comment]
@thomasdelhomenie  proposed PR ( https://github.com/exodev/platform/pull/93 ) to map  property to "exo.jcr.bloomfilter.enabled".

**Comment by aboughzela** [comment]
@fdrouet update the properties configuration name to allow the disable  bloom filter on a single workspace.
like gatein.jcr.bloomfilter.system.enabled for system workspace,  gatein.jcr.bloomfilter.social.enabled for social workspace ...

File: commons-extension-webapp/src/main/webapp/WEB-INF/conf/commons-extension/jcr/repository-configuration.xml [hunk 2]
```diff
@@ -175,13 +177,14 @@
...
 							<property name="db-tablename-suffix" value="PWORK"/>
+							<property name="acl-bloomfilter-enabled" value="${gatein.jcr.bloomfilter.portal-work.enabled:true}"/>
 						</properties>
 						<value-storages>
 							<value-storage id="portal-work"
                                            class="org.exoplatform.services.jcr.impl.storage.value.fs.TreeFileValueStorage">
 								<properties>
 									<property name="path" value="${gatein.jcr.storage.data.dir}/portal-work${container.name.suffix}"/>
-									<property name="enabled" value="${gatein.jcr.storage.enabled:true}"/>
+									<property name="enabled" value="${gatein.jcr.storage.portal-work.enabled:true}"/>
```
Comment (fdrouet) [inline commons-extension-webapp/src/main/webapp/WEB-INF/conf/commons-extension/jcr/repository-configuration.xml]: Why did you changed this property which is not related to bloomfilter ?

Comment (aboughzela) [inline commons-extension-webapp/src/main/webapp/WEB-INF/conf/commons-extension/jcr/repository-configuration.xml]: Thx for review,  revert this change.
